src/scenario/mod.rs

```diff
@@ -91,6 +91,9 @@ pub(crate) struct Scenario {
 
     #[serde(with = "required")]
     pub get_purl_details: Option<String>,
+
+    #[serde(with = "required")]
+    pub get_recommendations: Option<String>,
```

Comment:
Why don't we use Vec here (JSON array), like the endpoint expects? I think it would make more sense as we can then model calls better